weasyl/controllers/moderation.py

```diff
@@ -195,16 +196,20 @@ def modcontrol_edituserconfig_(request):
 @moderator_only
 @token_checked
 def modcontrol_copynotetostaffnotes_post_(request):
-    form = request.web_input(targetuserid=None, targetusername=None, notecontent=None,
-                             notedatetime=None, notesubject=None)
+    form = request.web_input(noteid=None)
 
-    staffnotetitle = ''.join(("Received note from ", form.targetusername, ", dated ", form.notedatetime,
-                              ', and the subject was: "', form.notesubject, '".'))
+    notedata = note.select_view(request.userid, int(form.noteid))
+
+    staff_note_title = "Received note from {sender}, dated {date}, and the subject was: \"{subj}\".".format(
```

Comment:
The mix of omitted and included articles is a bit odd. How about:

> Received note from {sender}, dated {date}, with subject “{subject}”

(Smart quotes there to avoid escaping and because they look nice; add `# encoding: utf-8` and make it a `unicode` literal.)